dev_tools/nyaml2nxdl/nyaml2nxdl_backward_tools.py

```diff
@@ -332,6 +333,51 @@ def clean_and_organise_text(self, text, depth):
 
         return text
 
+    def check_and_handle_doc_xref_and_other_doc(self, text, indent):
+        """Check for xref doc which comes as a block of text.
+
+        The doc part bellow is the example how xref comes:
+        '''
+        This concept is related to term `<term>`_ of the <spec> standard.
+        .. _<term>: <url>
+        '''
+        converter as
+        '''
+        <indent>  "xref:
+        <indent>    xpec: <value>
+        <indent>    erm: <value>
+        <indent>    url: <value>"
+        '''
+
+        Parameters
+        ----------
+        text: str
+            plain text.
+        Returns
+        -------
+        str
+            return part of doc as formatted
+        """
+
+        xref_key, spec_key, term_key, url_key = ("xref", "spec", "term", "url")
+        spec, term, url = (None, None, None)
+        matches = re.search(
+            r"This concept is related to term `([^`]+)`_ of the"
+            r" ([^\s]+) standard\.\s*\.\. _([^:]+): ([^\s]+)",
```

Comment:
```suggestion
            r"This concept is related to term `([^`:]+)`_ of the"
            r" ([^\s]+) standard\.\s+\.\. _\1: ([^\s]+)",
```

There should be *at least* one whitespace character after the end of the sentence. I also introduced that the terms need to match.

Reply:
LGTM

Reply:
addressed.